baselines/moon/moon/conf/cifar10.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+
+num_clients: 10
+num_epochs: 10
+fraction_fit: 1.0
+batch_size: 64
+learning_rate: 0.01
+mu: 5
+temperature: 0.5
+alg: moon
+seed: 0
+server_device: cpu
+num_rounds: 100
+
+client_resources:
+  num_cpus: 4
+  num_gpus: 1
```

Comment:
VRAM utilisation is fairly low (~900MB on my machine) for the CIFAR-10 model. I'm wondering if you could leave a lower value by default (e.g. `num_gpus=0.2`) to at least allow for 5 clients to run concurrently in the same GPU. More info on how to set `num_gpus` and `num_cpus` can be found in our docs: https://flower.dev/docs/framework/how-to-monitor-simulation.html. Adjusting these `client_resources` does make a big impact on how fast your experiments run.